roles/openshift_master/tasks/main.yml

```diff
@@ -25,7 +25,7 @@
       cluster_method: "{{ openshift_master_cluster_method | default(None) }}"
       cluster_hostname: "{{ openshift_master_cluster_hostname | default(None) }}"
       cluster_public_hostname: "{{ openshift_master_cluster_public_hostname | default(None) }}"
-      debug_level: "{{ openshift_master_debug_level | default(openshift.common.debug_level) }}"
+      debug_level: "{{ lookup('oo_option', 'openshift_master_debug_level') | default(openshift.common.debug_level, True) }}"
```

Comment:
Ah, please ignore my comment above about already setting openshift_master_debug level...  instead we should be setting openshift_master_debug_level in the playbooks, and keeping the role using the openshift_master_debug_level variable.